docs/apis-tools/console-api/tutorial.md

```diff
@@ -0,0 +1,263 @@
+
+```
+const options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+```
+
+6. Call the clients endpoint, process the results from the API call, emit the clients to output, and emit an error message from the server if necessary:
+
+```
+try {
+    // Call the clients endpoint.
+    const response = await axios(options);
+
+    // Process the results from the API call.
+    const results = response.data;
+
+    // Emit clients to output.
+    results.forEach(x => console.log(`Name: ${x.name}; ID: ${x.clientId}`));
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+```
+
+7. In your terminal, run `npm run cli admin list` for a list of your existing clients.
+
+:::note
+This `list` command is connected to the `listClients` function at the bottom of the `administration.js` file, and executed by the `cli.js` file. While we will view, create, and delete clients in this tutorial, you may add additional arguments depending on the API calls you would like to make.
+:::
+
+If you have any existing clients, the `Name: {name}; ID: {Id}` will now output. If you have an invalid API name or action name, or no arguments provided, or improper/insufficient credentials configured, an error message will output as outlined in the `cli.js` file.
+
+## POST a client
+
+To create a new client, you will follow similar steps as outlined in your [GET request] (#get-clientid) above:
+
+1. Edit the `addClient` function, incorporate the access token, and add your settings in the `.env` file.
+
+```
+async function addClient([clientName])
+const accessToken = await getAccessToken();
+
+  const administrationApiUrl = process.env.ADMINISTRATION_API_URL;
+  const clusterId = process.env.CLUSTER_ID;
+```
+
+2. Adjust your API endpoint to add a new client to a cluster:
+
+```
+const url = `${administrationApiUrl}/clusters/${clusterId}/clients`;
+```
+
+3. When configuring your API call, issue a POST request, and add a body containing information for the new client:
+
+```
+  var options = {
+    method: "POST",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    },
+    data: {
+      clientName: clientName
+    }
+  };
+```
+
+4. Call the `add` endpoint and process the results from the API call:
+
+```
+const response = await axios(options);
+const newClient = response.data;
+```
+
+5. Emit the new client to output. While different from this example, you will likely want to capture the `clientSecret` property from the response, as this cannot be displayed again:
+
+```
+console.log(
+      `Client added! Name: ${newClient.name}. ID: ${newClient.clientId}.`
+    );
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+```
```

Comment:
❤️ I love this re-write -- includes all lines of code, but breezes through the duplicates a little quicker than the first time, and highlights the changes. 💯